src/components/select/bl-select.ts

```diff
@@ -292,14 +291,16 @@ export default class BlSelect<ValueType extends FormValue = string> extends Form
     const inputSelectedOptions = html`<ul class="selected-options">
       ${this._selectedOptions.map(item => html`<li>${item.textContent}</li>`)}
     </ul>`;
-    const removeButton = html`<bl-button
-      class="remove-all"
-      size="small"
-      variant="tertiary"
-      kind="neutral"
-      icon="close"
-      @click=${this._onClickRemove}
-    ></bl-button>`;
+    const removeButton = this.clearable
```

Comment:
Multiple selects should always be clearable.

Reply:
Why is that?

Reply:
There could be cases like an option is selected by default and disabled, thus cannot be de-selected. If we display clear button it will clear that too. `clearable` prop should be fully optional.

Reply:
Apparently it's because of benchmarks: https://github.com/Trendyol/baklava/issues/608#issuecomment-1571508124

Also it's introducing another breaking change while fixing one. It should at least default to true.

Reply:
Oh, I though you meant the input, sorry. 

I don't quite see how this case would solve it. Since the data flow is unidirectional, developer could set required items back on clear. (Assuming there aren't any bugs with updating the value.)

It comes to my initial question actually. Should the design team decide when to render the clear button or the user?

If we area going with clearable prop (which I think is superior option for DX but not sure on design aspect of things.) it's default status should act like before:
- True by default on multiple select. User can provide false if needed.
- False by default on single select. User can provide true if needed.

Reply:
Hey @buseselvi, do you have any comments on this?

Reply:
I think we should keep multiple select as clearable true since there is no breaking change and it is more accurate in terms of UX.